app/src/main/java/com/abisoft/todocompose/model/TodoItemPost.kt

```diff
@@ -0,0 +1,28 @@
+package com.abisoft.todocompose.model
+
+import com.google.gson.annotations.SerializedName
+
+data class TodoItemPost(val status: String, val element: TodoItemNetwork, val revision:Int)
+
+data class TodoItemNetwork (
+    val id: String,
+    val text: String,
+    val importance: ImportanceNetwork,
+    val deadline: Long?,
+    val done: Boolean,
+    val color: String? = null,
+    @SerializedName("created_at")
```

Comment:
nit: I prefer to specify SerializedName for each property to make it more unified